fuel-core/src/cli/run.rs

```diff
@@ -148,3 +162,23 @@ pub async fn exec(command: Command) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+// Attempt to load the consensus key from cli arg first, otherwise check the env.
+fn load_consensus_key(
+    cli_arg: Option<String>,
+) -> anyhow::Result<Option<Secret<[u8; 32]>>> {
+    let secret_string = if let Some(cli_arg) = cli_arg {
+        warn!("Consensus key configured insecurely using cli args. Consider setting the {} env var instead.", CONSENSUS_KEY_ENV);
```

Comment:
Should we simply not support this if it's insecure?

Reply:
It's useful for dev testing when we do multinode setups